### PR TITLE
Chore: turndown-plugin-gfm: Fix package name in README

### PR DESCRIPTION
**Pull request by awxiaoxian2020**

### CODE REVIEW

**Comment by awxiaoxian2020** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by awxiaoxian2020** [comment]
recheck

File: packages/turndown-plugin-gfm/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ turndown-plugin-gfm is a suite of plugins which can be applied individually. The
 So for example, if you only wish to convert tables:
 
 ```js
-var tables = require('turndown-plugin-gfm').tables
+var tables = require('joplin-turndown-plugin-gfm').tables
```
Comment (personalizedrefrigerator) [inline packages/turndown-plugin-gfm/README.md]:
```suggestion
var tables = require('@joplin/turndown-plugin-gfm').tables
```

See

https://github.com/laurent22/joplin/blob/dev/packages/turndown-plugin-gfm/package.json#L2C12-L2C20

**Review by personalizedrefrigerator — CHANGES_REQUESTED** [review]
Thank you for the fix!

Be aware that we have a [script that automatically generates release notes from commit titles](https://github.com/laurent22/joplin/blob/f8a25cb275a0beb9ba3ef9bf4271d38947449f42/packages/tools/git-changelog.ts#L358). As such, a better name for this pull request might be `Chore: turndown-plugin-gfm: Fix package name in README` (the `Chore:` part prevents it from showing up in Joplin's release notes).

**Comment by awxiaoxian2020** [comment]
> Thank you for the fix!
> 
> Be aware that we have a [script that automatically generates release notes from commit titles](https://github.com/laurent22/joplin/blob/f8a25cb275a0beb9ba3ef9bf4271d38947449f42/packages/tools/git-changelog.ts#L358). As such, a better name for this pull request might be `Chore: turndown-plugin-gfm: Fix package name in README` (the `Chore:` part prevents it from showing up in Joplin's release notes).

Could you update the contributing guide to give a tip for other contributors? :)

**Comment by laurent22** [comment]
> Could you update the contributing guide to give a tip for other contributors? :)

It's all in the pull request description, when you create it:

> Please prefix the title with the platform you are targetting:
> 
> Here are some examples of good titles:
> 
> - Desktop: Resolves #123: Added new setting to change font
> - Mobile, Desktop: Fixes #456: Fixed config screen error
> - All: Resolves #777: Made synchronisation faster
> 
> And here's an explanation of the title format:
> 
> - "Desktop" for the Windows/macOS/Linux app (Electron app)
> - "Mobile" for the mobile app (or "Android" / "iOS" if the pull request only applies to one of the mobile platforms)
> - "CLI" for the CLI app
> 
> If it's two platforms, separate them with commas - "Desktop, Mobile" or if it's for all platforms, prefix with "All".
> 
> If it's not related to any platform (such as a translation, change to the documentation, etc.), simply don't add a platform.
> 
> Then please append the issue that you've addressed or fixed. Use "Resolves #123" for new features or improvements and "Fixes #123" for > bug fixes.
> 
> AND PLEASE READ THE GUIDE: https://github.com/laurent22/joplin/blob/dev/CONTRIBUTING.md

**Comment by laurent22** [comment]
Thanks for the fix!

**Pull request merged**